crates/rpc/rpc-builder/src/error.rs

```diff
@@ -62,3 +62,15 @@ impl RpcError {
         }
     }
 }
+
+impl PartialEq for RpcError {
+    fn eq(&self, other: &Self) -> bool {
+        match (self, other) {
+            (
+                RpcError::AddressAlreadyInUse { kind: kind1, error: error1 },
+                RpcError::AddressAlreadyInUse { kind: kind2, error: error2 },
+            ) => kind1 == kind2 && error1.raw_os_error() == error2.raw_os_error(),
+            _ => false,
+        }
+    }
+}
```

Comment:
we don't need this and there's a reason why this does not exist on io error and json rcperror

also this implementation is not accurate, so please remove entirely.